utils/errlock/errlock.go

```diff
@@ -5,11 +5,17 @@ import (
 	"io"
 	"os"
 	"path"
+
+	"github.com/Fantom-foundation/go-opera/utils/caution"
 )
 
 // Check if errlock is written
 func Check() error {
-	locked, reason, eLockPath, _ := read(datadir)
+	locked, reason, eLockPath, err := read(datadir)
+	if err != nil {
+		return fmt.Errorf("Node isn't allowed to start due to an error reading lock file. Please fix the issue and then delete file \"%s\". Error message:\n%w", eLockPath, err)
```

Comment:
please check: The message, I guess, should be updated as follows:
```suggestion
		return fmt.Errorf("Node isn't allowed to start due to an error reading the lock file "%s". Please fix the issue. Error message:\n%w", eLockPath, err)
```